bpfd/src/bpf.rs

```diff
@@ -64,10 +63,6 @@ impl<'a> BpfManager<'a> {
         direction: Option<Direction>,
         path: &str,
     ) -> Result<(), anyhow::Error> {
-        debug!(
```

Comment:
any reason for removing all this logging?

Reply:
yeah the values it was logging disappeared as I was updating the API so deleted it to remove compiler erros! can look at adding it back 😆